src/i18n/en-US/general.ts

```diff
@@ -0,0 +1,5 @@
+const general = {
+  welcome: 'Welcome to EASi'
```

Comment:
For this initial pr, can you change this to be nested under home? That way home page text can be on the same level as welcome and it gives more context. I'd also suggest doing the whole home page in i18n since there's not that much text (not the banner components).